game/scripts/vscripts/modifiers/modifier_boss_capture_point.lua

```diff
@@ -120,6 +123,7 @@ function modifier_boss_capture_point:OnIntervalThink()
     -- Point is being captured by a team
     captureTick = self.thinkInterval
   end
+  captureTick = captureTick * math.max(1, heroMultiplier/2)
```

Comment:
Doesn't this only increase the capture rate for 3 heroes or more?

Reply:
yeah, i guess it really shoulda been `1 + hm/2`

Reply:
`0.5 + hm/2` 1 would make the first hero capture at 1.5x speed. :P